.github/workflows/publish-docsite.yaml

```diff
@@ -128,3 +128,9 @@ jobs:
           # re-enable PR comment bot
           COMMIT_INFO_SHA: ${{github.event.pull_request.head.sha}}
 
+      # - name: Rollback For Failed Cypress Tests
+      #   if: ${{ failure() }}
+      #   id: rollback
+      #   uses: ./.github/actions/get_penultimate_release
+      #   with:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Reason this is commented out?

Reply:
Because that action calls a workflow that doesn't exist until this is on main so all of that will be updated in another PR to turn it on.